### ISSUE TITLE
eval fails

### ISSUE THREAD

**Issue by sakshi79** [geyan21/ManiFlow_Policy#8]
Hi, I ran robotwin pointcloud version for different tasks. The policy is not able to learn. The mean success rate is close to zero (-ve) and all eval attempts fail with different seeds. Is there something I might be missing?

**Comment by geyan21** [comment]
Hi, could you please provide your training curves (e.g., loss and action MSE error) as well as the evaluation videos?

**Comment by akajiejie** [comment]
You can try visualizing the point cloud to check whether the segmentation of the point cloud is reasonable, which greatly affects the success rate of point cloud processing.

**Comment by geyan21** [comment]
The point cloud quality in RoboTwin should be fine. Regarding what @akajiejie said, one thing to note is ManiFlow also works well without cropping. Just make sure there are enough points to cover the target objects.

**Comment by sakshi79** [comment]
> Hi, could you please provide your training curves (e.g., loss and action MSE error) as well as the evaluation videos?

Hi, thank you for your response. The loss and mse curves seem fine, I am worried about the test_mean_score being pretty low.
What's the right metric for success rate here?

<img width="2130" height="631" alt="Image" src="https://github.com/user-attachments/assets/8bc2700b-ac9e-4639-a397-f7aa6c27965b" />

**Comment by sakshi79** [comment]
Also, found this in training code, wonder why is it defined like that? 

<img width="652" height="68" alt="Image" src="https://github.com/user-attachments/assets/962f8587-d1fd-439f-803d-7b7bc0cd46a8" />
